modules/best-practices-rhacm-discovered-vm.adoc

```diff
@@ -12,7 +12,7 @@ You can take several actions to improve your experience and chance of success wh
 
 Protecting the VM when using MTV, the {product-title} web console, or a custom VM:: Because automatic labeling is not currently available, the application owner must manually label the components of the VM application when using MTV, the {product-title} web console, or a custom VM.
 +
-After creating the VM, apply a common label to the following resources associated with the VM: `VirtualMachine`, `DataVolume`, `PersistentVolumeClaim`, `Service`, `Route`, `Secret`, `ConfigMap`, `VirtualMachinePreference`, and `VirtualMachineInstancetype`. Do not label virtual machine instances (VMIs) or pods; {VirtProductName} creates and manages these automatically.
+After creating the VM, apply a common label to the following resources associated with the VM: `VirtualMachine`, `DataVolume`, `PersistentVolumeClaim`, `Service`, `Route`, `Secret`, `ConfigMap`, `VirtualMachinePreference`, `VirtualMachineInstancetype`, and `ControllerRevision`. Do not label virtual machine instances (VMIs) or pods; {VirtProductName} creates and manages these automatically.
```

Comment:
```suggestion
After creating the VM, apply a common label to the following resources associated with the VM: `VirtualMachine`, `DataVolume`, `PersistentVolumeClaim`, `Service`, `Route`, `Secret` and `ConfigMap`. If an instance type or preference is used by the VM then the `ControllerRevision` copy of these objects referenced by the spec or status of the VM should also be labelled. Do not label virtual machine instances (VMIs) or pods; {VirtProductName} creates and manages these automatically.
```

Reply:
Thanks for this. I made some minor changes so that it fits our writing guidelines, but this is updated now. PTAL.